.azure-devops/full-pipeline.yml

```diff
@@ -12,3 +12,21 @@ jobs:
       - template: ./steps/devcerts.yml
       - template: ./steps/edgewebview.yml
       - template: ./steps/test.yml
+
+  - job: Mac
+    pool:
+      name: OPX Mac Dogfood Pool
+    steps:
+    - template: ./steps/npm-install.yaml
+    - template: ./steps/build.yaml
+    - template: ./steps/run-tests.yaml
+    - template: ./steps/cleanup.yaml
```

Comment:
Is this a mac exclusive task?

Reply:
Sort of.  It's needed on the mac because we're reusing the hardware repeatedly.  The other two cases are single use vms that get tossed when the job is done.